api/onix.py

```diff
@@ -224,6 +225,12 @@ def parse(cls, file, data_source_name, default_medium=None):
             issued = None
             if publishing_date:
                 issued = dateutil.parser.isoparse(publishing_date)
+                if issued.tzinfo is None:
+                    cls._logger.warning(
+                        "Publishing date {} does not contain timezone information. Assuming UTC."
+                        .format(publishing_date)
+                    )
+                    issued = issued.replace(tzinfo=pytz.UTC)
```

Comment:
This looks good but I would suggest to use [`datetime_helpers.to_utc`](https://github.com/ThePalaceProject/circulation-core/blob/main/util/datetime_helpers.py#L30) to convert datetime to UTC.